data/json/items/melee/bludgeons.json

```
@@ -935,5 +934,39 @@
     "cutting": 23,
     "price": "160 USD",
     "qualities": [ [ "HAMMER", 2 ] ]
+  },
+  {
+    "id": "qstaff_retracted",
+    "type": "GENERIC",
+    "category": "weapons",
+    "looks_like": "baton",
+    "weapon_category": [ "BATONS" ],
+    "name": { "str": "extendable quarterstaff (retracted)", "str_pl": "extendable quarterstaves (retracted)" },
+    "description": "A custom-made telescoping steel quarterstaff, capable of shrinking down to the size of a baton for easy storage. Its hollow nature makes it more lightweight but less sturdy than a more traditional staff. It is currently retracted.",
```

Also goofy ahh spaces rule
```suggestion
    "description": "A custom-made telescoping steel quarterstaff, capable of shrinking down to the size of a baton for easy storage.  Its hollow nature makes it more lightweight but less sturdy than a more traditional staff.  It is currently retracted.",
```